.ci/appveyor.yml

```diff
@@ -20,6 +21,14 @@ cache:
   - "C:\\Users\\appveyor\\AppData\\Local\\coala-bears\\coala-bears"
   - "C:\\Users\\appveyor\\AppData\\Roaming\\nltk_data"
 
+# Workaround Appvayor bug:
+# Ignore YAMLLintBear
```

Comment:
this is a bug in yamllintbear .. it enable the 'unbreakable-words' rule; see http://yamllint.readthedocs.io/en/latest/rules.html?highlight=allow-non-breakable-words#module-yamllint.rules.line_length